src/qcrypto/simbasics.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import dataclasses
+
+@dataclasses.dataclass
+class Qubit:
+    """
+    Data class which represents the qubit and its quantum state.
+    Quantum state is given by:
+    |psi> = cos(theta/2) |0> + e^{i*phi} * sin(theta/2) |1>
```

Comment:
$|\psi> = \cos(\theta/2) |0> + e^{i*\phi}  \sin(\theta/2) |1>$ 

Maybe we can give the LaTeX version as well for future rendering?